src/test/java/uk/gov/hmcts/reform/bulkscanprocessor/validation/OcrValidatorTest.java

```diff
@@ -283,7 +283,7 @@ public void should_throw_an_exception_if_service_responded_with_error_response()
         // then
         assertThat(err)
             .isInstanceOf(OcrValidationException.class)
-            .hasMessageContaining("Error!");
+            .hasMessageContaining("OCR validation service returned OCR-specific errors");
```

Comment:
might be good to also assert that it DOES NOT contain "Error!" anymore. Ideally with a comment or assertj's description.
Or change "Error!" to something like "OCR error that should not be included in exception message" ;)

Reply:
That would be excessive - there's a whole world of stuff that we don't want in this exception.